menagerie/concept-shapes/tests/test_wp_notes_subsumption.py

```
@@ -0,0 +1,169 @@
+
+    def test_notes_only_difference_discharges_via_wp_abstraction(self):
+        """wp texts match; only notes differs.  Should discharge (notes is doc-only)."""
+        shared_wp = "lossless source wrapper; the source bytes are recoverable"
+        concept = _make_spec(wp=shared_wp)
+        lang = _make_spec(wp=shared_wp, notes="Language-specific implementation detail.")
+        result = mlm.try_structural_subsumption(lang, concept)
+        self.assertIsNotNone(
+            result,
+            "notes-only difference should discharge via wp-abstraction; got None (gap)"
+        )
+        method, pre_relaxed, wp_abstracted = result
```

Ruff (0.15.12)

[warning] Unpacked variable `wp_abstracted` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)